src/llm/llm.controller.ts

```
@@ -18,16 +22,72 @@ export class LlmController {
+          });
         return ({
             extract : extract,
             embedding : embedding,
+            langchain : chunks,
             moderation : moderation.results[0].flagged,
+            qdrant : storeVector,
             // embedding : embedding,
         })
+
+    }
+    @Get('progress')
```

ESLint

[error] Delete `··`

(prettier/prettier)